modules/test/playwright/tests/change-tracking-web/ctCollectionPublish.spec.ts

```diff
@@ -0,0 +1,68 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {expect, mergeTests} from '@playwright/test';
+
+import {changeTrackingPagesTest} from '../../fixtures/changeTrackingPagesTest';
+import getRandomString from '../../utils/getRandomString';
+import {journalPagesTest} from '../journal-web/fixtures/journalPagesTest';
+
+export const test = mergeTests(journalPagesTest, changeTrackingPagesTest);
+
+let journalName;
+let text;
+
+test('Cannot publish an empty ctCollection', async ({
+	changeTrackingPage,
+	ctCollection,
+	journalEditArticlePage,
+	journalPage,
+	page,
+}) => {
+	journalName = getRandomString();
+	text = getRandomString();
+
+	await journalPage.goto();
+
+	await changeTrackingPage.changeToProduction(ctCollection);
+
+	await journalPage.goToCreateArticle();
+
+	await journalPage.createBasicArticle(journalName, text);
+
+	await changeTrackingPage.changeToPublication(ctCollection);
+
+	await journalEditArticlePage.editAndPublishExistingBasicArticle(
+		journalName
+	);
+
+	await page.getByRole('button', {name: ctCollection.name}).click();
+
+	await page.getByRole('menuitem', {name: 'Work on Production'}).click();
+
+	await expect(page.locator('.change-tracking-indicator-title')).toHaveText(
+		'Production'
+	);
```

Comment:
Can we use `changeTrackingPage.changeToProduction(ctCollection)` here?

Reply:
We can't, it gives an error due to not finding a dialog. There's a dialog acceptance into the function, in playwright it only needs to be called once, the second time it will be accepted even if we do not call it. To avoid the error I added the function code, but omitted the dialog part, so it wouldn't be searched twice

Reply:
@gislayne-vitorino can you check if we can use `apiHelpers.headlessChangeTracking.checkoutCTCollection` to switch between publication and production? That way we don't have to deal with the dialog.